perfmetrics/scripts/testing_on_gke/examples/run-gke-tests.sh

```diff
@@ -95,6 +92,7 @@ function printHelp() {
   echo "instance_id=<string, not containing spaces, representing unique id for particular test-run e.g. \"${DEFAULT_INSTANCE_ID}\""
   echo "workload_config=<path/to/workload/configuration/file e.g. /a/b/c.json >"
   echo "output_dir=</absolute/path/to/output/dir, output files will be written at output_dir/fio/output.csv and output_dir/dlio/output.csv>"
+  echo "force_update_gcsfuse_code=<true|false, to force-update the gcsfuse-code to given branch if gcsfuse_src_dir has been set. Default=\"${DEFAULT_FORCE_UPDATE_GCSFUSE_CODE}\">"
```

Comment:
Where is DEFAULT_FORCE_UPDATE_GCSFUSE_CODE defined ?

Reply:
Already defined on line number 65. This change doesn't introduce this variable, only publishes it. It was defined weeks back as it was needed for the automated job.